src/intro-to-c/debugging/gdb-intro.md

```diff
@@ -185,9 +199,17 @@ Breakpoint 1, main () at my_file.c:6
 (gdb)
 ```
 
+> The `break` command lets you set breakpoints, specific points where you want your program to stop while it's running. A breakpoint gives you a location from which you can choose how and where you want to proceed, whether line-by-line or multiples lines at a time.
+
+> Here, we pass `break` a function name as an argument, but you can also pass it a line number (e.g., `break 6`). If you wanted to be more specific, you could pass it a location written as a file name, colon, and either a function name or a line number (e.g., `break my_file.c:main`, `break my_file.c:6`).
```

Comment:
Do you think I should I keep this sentence? It's more useful to know when you're compiling multiple source files together, but I think you typically compile single files in Intro to C.

Reply:
I think this is good extra information. No harm in keeping it, since this lesson is placed towards the end of the lesson list. A student may be jumping into CS1 at this point.